### DISCUSSION TITLE
Is it advisable to migrate from version 5.2.x to 6.x?

### DISCUSSION THREAD

**Discussion by deh4g** [storybookjs/storybook#15342] [Help] [answered]
Dear Storybook community,

Is it advisable to migrate from version 5.2.x to 6.x without first migrating to 5.3 ?

If there are issues, kindly elaborate.

Many thanks.

**Comment by shilman** [comment, marked as answer]
Yes, I'd follow the SB 6 migration guide to go directly from 5.2 to 6.3: https://storybook.js.org/blog/storybook-6-migration-guide/

Then I'd skim `MIGRATION.md` if anything goes wrong, or just to get a sense if there are changes that you might want to know about. https://github.com/storybookjs/storybook/blob/next/MIGRATION.md

**Reply by deh4g** [reply]
Appreciate the quick reply, Michael. Thanks.
